Package.swift

```diff
@@ -25,8 +25,8 @@ let package = Package(
       ),
     ],
     dependencies: [
-      .Package(url: "https://github.com/IBM-Swift/Swift-cfenv.git", majorVersion: 2),
-      .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1),
+      .Package(url: "https://github.com/IBM-Swift/CloudConfiguration", majorVersion: 1),
```

Comment:
This should be .Package(url: "https://github.com/IBM-Swift/CloudConfiguration**.git**", majorVersion: 1), otherwise SPM will try and download separate packages for this and SwiftMetrics (which uses .git)